acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/PageHelper.java

```diff
@@ -63,22 +63,74 @@ private WebElement findElement(final Function<By, List<WebElement>> findElements
         }
     }
 
+    public WebElement findWhenStable(final Supplier<WebElement> findWebElementFunction) {
```

Comment:
I'm struggling with this name. I'm not sure about the "Stable" part. Let's have a chat about it.

Reply:
Now removed - as agreed, I've replace this method with an overload of `PageHelper#findElement(java.util.function.Function<org.openqa.selenium.By,java.util.List<org.openqa.selenium.WebElement>>, org.openqa.selenium.By, java.util.function.Function<org.openqa.selenium.WebElement,java.lang.Boolean>)`.